modules/ssl/ssl_engine_vars.c

```diff
@@ -430,10 +430,54 @@ const char *ssl_var_lookup(apr_pool_t *p, server_rec *s,
     return result ? result : "";
 }
 
+#ifdef HAVE_OPENSSL_ECH
+/* Extract ECH status variable from SSL object 'ssl' */
+static const char *ssl_var_lookup_ech_status(apr_pool_t *p, const char *var, SSL *ssl)
+{
+    char *inner_sni = NULL, *outer_sni = NULL;
+    int echrv;
+    const char *result = NULL;
+
+    if (ssl == NULL)
+        return "";
```

Comment:
If nothing was found leave it at `NULL`.
```suggestion
        return result;
```

Reply:
sure